tests/phpunit/class-wc-stripe-blocks-support-test.php

```
@@ -0,0 +1,110 @@
+		}
+
+		$property->setValue( WC_Stripe::get_instance(), $gateway );
+	}
+
+	/**
+	 * Restores the original main gateway and removes the empty-gateways filter.
+	 *
+	 * @return void
+	 */
+	public function tearDown(): void {
+		$property = new ReflectionProperty( WC_Stripe::class, 'stripe_gateway' );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '41', column '19'). (undefined)

(MissingImport)